### PR TITLE
I know this is the wrong branch but we gotta get this change in quick!! Reroutes notification click to correct url.

### PR DESCRIPTION
**Pull request by nlumiere**

### CODE REVIEW

**Review by showermat — APPROVED** [review]
The hardcoded UUID makes me feel weird, but it was already there and I'm going to trust you all.

File: proj/src/main/webapp/notifications.js [hunk 1]
```diff
@@ -101,14 +101,21 @@ function sendNotificationForChat(message) {
     const tag = sessionStorage.activeChatTag;
     const chatId = sessionStorage.activeChatId;
     const name = message.senderDisplay;
-    notificationBody = name+' has sent a message:\n\n'+message.content
+    notificationBody = name+' has sent a message:\n\n'+message.content;
+    var url;
+    if(window.location.href.includes("https://8080")){
+        url = 'https://8080-9f73b32e-6ccb-426e-8d6e-c03d6f324666.us-west1.cloudshell.dev/static/chat.html';
```
Comment (aokeke) [inline proj/src/main/webapp/notifications.js]: why not just reroute to window.location.href? Won't this mean other people will reroute to your dev server?

**Review by aokeke — APPROVED** [review]
LGTM with reservations

**Review by aokeke — APPROVED** [review]
LGTM

**Pull request merged**